.stestr.blacklist.functional

```diff
@@ -0,0 +1,2 @@
+octavia_proxy.tests.functional.api.v2.test_load_balancer.TestLoadBalancer.test_create_with_tags
+octavia_proxy.tests.functional.api.v2.test_listener.TestListener.test_create_with_tags
```

Comment:
you can clear this file now

Reply:
@anton-sidelnikov ELBv2 tag support is still missing and funct tests are are using ELBv2:
```
openstack.exceptions.BadRequestException: Attribute 'tags' not allowed in POST: Client Error for url: https://elb.eu-de.otc.t-systems.com/v2.0/lbaas/listeners, Attribute 'tags' not allowed in POST
```